src/config.ts

```diff
@@ -72,16 +72,16 @@ export default {
   },
   s3StorageRepo: {
     awsConfig: {
-      accessKeyId: getStringOption(process.env.FS_S3_ACCESS_KEY_ID, globalAwsIamAccessKeyId),
       apiVersion: '2006-03-01',
       region: getStringOption(process.env.FS_S3_REGION, globalAwsRegion),
-      secretAccessKey: getStringOption(
-        process.env.FS_S3_SECRET_ACCESS_KEY,
-        globalAwsIamAccessKeySecret,
-      ),
-      signatureVersion: 'v4',
-      sslEnabled: true,
```

Comment:
`signatureVersion` seems to have been removed in v3 of the sdk, but `sslEnabled` was renamed `tls`.
https://github.com/aws/aws-sdk-js-v3/blob/main/UPGRADING.md#:~:text=sslEnabled,to%20tls.
https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-s3/interfaces/s3clientconfig.html#tls

I can't see what the default behaviour is if omitted, but this should probably be set to true as it was before. i.e.:
```
tls: true,
```

Reply:
👍